test/test_integration_configurations.py

```diff
@@ -33,6 +35,7 @@ def setup_cli_params(self, env_vars, *params):
 
     def setUp(self):
         self.setup_cli_params({})
+        utils.delete_credentials()
```

Comment:
why do we need to delete credentials if these tests check the conjurrc integrity. All else doesn't really matter. No?